scrapy/core/http2/protocol.py

```diff
@@ -0,0 +1,191 @@
+import ipaddress
+import itertools
+import logging
+from collections import deque
+
+from h2.config import H2Configuration
+from h2.connection import H2Connection
+from h2.events import (
+    DataReceived, ResponseReceived, SettingsAcknowledged,
+    StreamEnded, StreamReset, WindowUpdated
+)
+from twisted.internet.protocol import connectionDone, Protocol
+from twisted.internet.ssl import Certificate
+
+from scrapy.core.http2.stream import Stream, StreamCloseReason
+from scrapy.http import Request
+
+LOGGER = logging.getLogger(__name__)
```

Comment:
:lipstick: 
```suggestion

logger = logging.getLogger(__name__)
```

Other occurrences of the logger would need be updated too.